.gitignore

```diff
@@ -1,12 +1,9 @@
-test/bundle.js
```

Comment:
This still needs to be ignored until #115 is merged. We could merge that first though.